Tasks/PackerBuildV1/src/taskParameters.ts

```diff
@@ -82,8 +83,13 @@ export default class TaskParameters {
                 this.deployScriptArguments = tl.getInput(constants.DeployScriptArgumentsInputName, false);
 
                 this.graphCredentials = this._getAzureADGraphCredentials(this.serviceEndpoint);
+                
             }
-
+            
+            console.log(tl.loc("ResolvingPackerVersionInput"))
+            this.packerVersionString = tl.getInput(constants.PackerVersionInputName);
+            console.log(tl.loc("ResolvedPackerVersion", this.packerVersionString));
```

Comment:
I think you need not localize these and you can put this in tl.debug

Reply:
removed localization, system.debug=true already printing the parsed input value